ui/game/PieceGraveyard.js

```diff
@@ -60,12 +65,12 @@ module.exports = (
   function renderPiecesForColour(colour) {
     let pieces = [];
 
-    for (const pieceName in materialDiff[colour]) {
+    Object.keys(materialDiff[colour]).forEach((pieceName) => {
```

Comment:
This throws an error upon opening a game now.

```
TypeError: Cannot convert undefined or null to object
```

Maybe we can do something more null safe here?